backend/wargamebackend/auth/views.py

```
@@ -1,10 +1,41 @@
 from rest_framework import status
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from django.contrib.auth.models import User
+from auth.authentication import CookieJWTAuthentication
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from rest_framework.decorators import api_view
 from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework_simplejwt.views import (
+    TokenRefreshView,
+    TokenObtainPairView,
+    TokenRefreshView
```

Ruff (0.12.2)

Redefinition of unused `TokenRefreshView` from line 13

Remove definition: `TokenRefreshView`

(F811)